components/Article/index.js

```diff
@@ -12,35 +12,41 @@ function Article({ article }) {
   const { id, name, size, gender, price, image } = article;
 
   return (
-    <StyledArticle>
-      <StyledArticleImageContainer>
-        <StyledImage
-          variant="article"
-          src={image}
-          alt={`Ein Bild von ${name}`}
-          fill
-        />
-      </StyledArticleImageContainer>
-      <h3>{name}</h3>
+    <>
+      <StyledArticle>
+        <StyledArticleImageContainer>
+          <StyledImage
+            variant="article"
+            src={
+              image === ""
+                ? "https://res.cloudinary.com/depezzq0e/image/upload/v1670228543/placeholder-image_j7hueu.jpg"
+                : image
+            }
+            alt={`Ein Bild von ${name}`}
+            fill
+          />
+        </StyledArticleImageContainer>
+        <h3>{name}</h3>
 
-      <StyledList>
-        <StyledListItem>
-          Größe: <span>{size}</span>
-        </StyledListItem>
-        <StyledListItem>
-          Geschlecht:<span>{gender}</span>
-        </StyledListItem>
-        <StyledListItem>
-          Preis (Euro): <span>{price}</span>
-        </StyledListItem>
-      </StyledList>
-      <StyledDetailsLink
-        href={`/articleDetails/${id}`}
-        aria-label="Article Details"
-      >
-        <Svg variant={"details"}></Svg>
-      </StyledDetailsLink>
-    </StyledArticle>
+        <StyledList>
+          <StyledListItem>
+            Größe: <span>{size}</span>
+          </StyledListItem>
+          <StyledListItem>
+            Geschlecht:<span>{gender}</span>
+          </StyledListItem>
+          <StyledListItem>
+            Preis (Euro): <span>{price}</span>
+          </StyledListItem>
+        </StyledList>
+        <StyledDetailsLink
+          href={`/articleDetails/${id}`}
+          aria-label="Article Details"
+        >
+          <Svg variant={"details"}></Svg>
+        </StyledDetailsLink>
+      </StyledArticle>
+    </>
```

Comment:
- you can delete the react fragments `<></>` because the `<StyledArticle>` component encloses your children.
- _nitpicking little comment:_  you can remove the curly braces if the prop is a string, like you did in line 19. I would recommend you to use a consistent spelling. 

```suggestion
        <StyledList>
          <StyledListItem>
            Größe: <span>{size}</span>
          </StyledListItem>
          <StyledListItem>
            Geschlecht:<span>{gender}</span>
          </StyledListItem>
          <StyledListItem>
            Preis (Euro): <span>{price}</span>
          </StyledListItem>
        </StyledList>
        <StyledDetailsLink
          href={`/articleDetails/${id}`}
          aria-label="Article Details"
        >
          <Svg variant="details" />
        </StyledDetailsLink>
      </StyledArticle>
  
```